[2025-10-11T06:07:30.406+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-11T06:07:30.461+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline_flow.check_create_stg_tables scheduled__1998-08-16T00:00:00+00:00 [queued]>
[2025-10-11T06:07:30.476+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline_flow.check_create_stg_tables scheduled__1998-08-16T00:00:00+00:00 [queued]>
[2025-10-11T06:07:30.478+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-11T06:07:30.498+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): check_create_stg_tables> on 1998-08-16 00:00:00+00:00
[2025-10-11T06:07:30.511+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=9852) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-11T06:07:30.513+0000] {standard_task_runner.py:64} INFO - Started process 9854 to run task
[2025-10-11T06:07:30.512+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipeline_flow', 'check_create_stg_tables', 'scheduled__1998-08-16T00:00:00+00:00', '--job-id', '610', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dag.py', '--cfg-path', '/tmp/tmpb6fv0co0']
[2025-10-11T06:07:30.516+0000] {standard_task_runner.py:91} INFO - Job 610: Subtask check_create_stg_tables
[2025-10-11T06:07:30.601+0000] {task_command.py:426} INFO - Running <TaskInstance: pipeline_flow.check_create_stg_tables scheduled__1998-08-16T00:00:00+00:00 [running]> on host 321597a50810
[2025-10-11T06:07:30.736+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='oluwaseyiogundimu10@gmail.com' AIRFLOW_CTX_DAG_OWNER='seyi' AIRFLOW_CTX_DAG_ID='pipeline_flow' AIRFLOW_CTX_TASK_ID='check_create_stg_tables' AIRFLOW_CTX_EXECUTION_DATE='1998-08-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__1998-08-16T00:00:00+00:00'
[2025-10-11T06:07:30.737+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-11T06:07:30.754+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:07:30.843+0000] {connection.py:43} INFO - Table 'stg_users' in Postgres.
[2025-10-11T06:07:30.844+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:07:30.924+0000] {connection.py:43} INFO - Table 'users' in Postgres.
[2025-10-11T06:07:30.925+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:07:31.116+0000] {connection.py:43} INFO - Table 'stg_movies' in Postgres.
[2025-10-11T06:07:31.119+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:07:31.209+0000] {connection.py:43} INFO - Table 'movies' in Postgres.
[2025-10-11T06:07:31.210+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:07:31.298+0000] {connection.py:43} INFO - Table 'stg_ratings' in Postgres.
[2025-10-11T06:07:31.299+0000] {connection.py:22} INFO - Successfully connected to Postgres.
[2025-10-11T06:07:31.379+0000] {connection.py:43} INFO - Table 'ratings' in Postgres.
[2025-10-11T06:07:31.379+0000] {load.py:32} INFO - Tables exist.
[2025-10-11T06:07:31.381+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-11T06:07:31.382+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-11T06:07:31.399+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipeline_flow, task_id=check_create_stg_tables, run_id=scheduled__1998-08-16T00:00:00+00:00, execution_date=19980816T000000, start_date=20251011T060730, end_date=20251011T060731
[2025-10-11T06:07:31.454+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-10-11T06:07:31.493+0000] {taskinstance.py:3503} INFO - 3 downstream tasks scheduled from follow-on schedule check
[2025-10-11T06:07:31.496+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
